packaging/debian/control

```diff
@@ -0,0 +1,22 @@
+Source: connman-curses-git
+Maintainer: Javier López <m@javier.io>
+Section: net
+Priority: optional
+Build-Depends: debhelper  (>=9), libjson-c2, libjson-c-dev, autoconf (>=2.69), automake, libncurses5, libncurses5-dev, libdbus-1-dev, dh-exec (>= 0.12)
```

Comment:
libjson was renamed to libjson-c (https://github.com/json-c/json-c/issues/5) in debian/ubuntu libjson0 is a transitional package (who may disappear in future releases), libjson-c2 is the new package name assigned to libjson-c.